commit f5bd0ed47523b39aedb6bcc1f9c0754371159a77
Author: Rich Megginson <rmeggins at redhat.com>
Date:   Fri Sep 14 09:20:18 2012 -0600

    Ticket #461 - fix build problem with mozldap c sdk
    
    https://fedorahosted.org/389/ticket/461
    Reviewed by: rmeggins
    Fixed by: cgrzemba
    Branch: master
    Fix Description: mozldap does not define LDAP_MOD_OP so define it
    Platforms tested: RHEL6 x86_64
    Flag Day: no
    Doc impact: no

diff --git a/ldap/servers/slapd/pw.c b/ldap/servers/slapd/pw.c
index bfd48b1..4736e82 100644
--- a/ldap/servers/slapd/pw.c
+++ b/ldap/servers/slapd/pw.c
@@ -61,6 +61,9 @@
 #if defined( _WIN32 )
 #undef LDAPDebug
 #endif	/*  _WIN32 */
+#if defined( USE_MOZLDAP )
+#define LDAP_MOD_OP (0x0007)
+#endif /* USE_MOZLDAP */
 
 #include "slap.h"
